docs/framework/react/api/router/ParseRouteChildrenType.md

```diff
@@ -4,3 +4,9 @@ title: ParseRouteChildren type
 ---
 
 This type recursively parses a route's children and grandchildren into a single union of all possible routes.
+
+```tsx
+export type ParseRouteChildren = {
```

Comment:
this type does not exist anymore.
even more proof we should not document each internal type as they might... change or vanish

Reply:
Shall we keep a log of what doesn't exist anymore (and what shouldn't be exposed) and remove those in a following PR?
I'm currently keeping track of what's missing in #1231 to be filled in after this PR, let's track the stuff to be hidden from the user in an issue as well and fasttrack it before the new pages. Since it's just a matter of *deletes*, it ideally should take much time.

Reply:
yeah sure let's remove it in a separate PR. I just noticed it during review.